WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -112,7 +113,7 @@ public static boolean addOrUpdateBlog(String blogName, String xmlRpcUrl, String
             // deprecated
             blog.setMaxImageWidthId(0);
             blog.setRemoteBlogId(Integer.parseInt(blogId));
-            blog.setDotcomFlag(xmlRpcUrl.contains("wordpress.com"));
+            blog.setDotcomFlag(WPUrlUtils.isWordPressCom(xmlRpcUrl));
```

Comment:
I wonder if we should create a new DB version and migrate all sites' `DotComFlag`.